src/components/theme-switcher/user-dropdown-switcher/index.tsx

```diff
@@ -10,10 +10,7 @@ export default function UserDropdownDisclosureThemeSwitcher() {
 		<>
 			<DropdownDisclosureSeparatorItem />
 			<DropdownDisclosureListItem className={s.listItem}>
-				<label className={s.label} htmlFor="theme-switcher">
```

Comment:
Bug fix here: accidentally updated the theme switcher id below but forgot to match it to this label. The new `WithLabel` primitive should help to avoid that.